packages/directives/resize/index.ts

```diff
@@ -3,16 +3,20 @@ import {
   removeResizeListener,
 } from '@element-plus/utils/resize-event'
 
-import type { ObjectDirective } from 'vue'
+import type { ObjectDirective, DirectiveBinding } from 'vue'
+
+declare interface ResizeEl {
+  _handleResize: () => void
+}
 
 const Resize: ObjectDirective = {
-  beforeMount(el, binding) {
+  beforeMount(el: ResizeEl, binding: DirectiveBinding) {
     el._handleResize = () => {
```

Comment:
`el` is `HTMLElement`, how about
```suggestion
  beforeMount(el: HTMLElement, binding: DirectiveBinding) {
   (el as any)._handleResize = () => {
```

Reply:
HTMLElement does not exist _handleResize function

Reply:
```
(el as any) 
```
In this case, it's still any type.